src/config/common.js

```diff
@@ -48,6 +49,16 @@ export function getResolve(config, paths) {
         '.web.js', '.web.jsx', '.web.ts', '.web.tsx',
         '.js', '.json', '.jsx', '.ts', '.tsx',
       ],
+      alias: {
```

Comment:
I am not sure this is a good idea.
- Roadhog is not just for dva project.
- alias do simplify code, but is make vscode go to definition becomes diffcult.